reference/protocol.ksy

```diff
@@ -300,6 +298,8 @@ types:
         size: 64
       - id: sequence
         type: u8le
+      - id: timestamp
```

Comment:
timestamp replaces sequence, it is not a sibling of it

Reply:
Ah, I was confusing it with other areas where we left them as siblings then. Thanks, resolved!

Reply:
The type does change either. It remains u8le.

Reply:
Ah, thanks...resolved!